e2e/ansible/playbooks/hyperconverged/test-k8s-memleak/memleak-cleanup.yml

```diff
@@ -8,8 +8,16 @@
   delegate_to: "{{groups['kubernetes-kubeminions'].0}}"
   register: result
 
+- name: Obtain the pvc name
+  shell: source ~/.profile; kubectl get pvc -n {{ namespace }} | awk '{print $1}'
```

Comment:
Use no-headers if pvc.stdout is resued. Also, a better approach is to derive the PV ID from the PVC